### PR TITLE
added readme file and slider component

### PR DESCRIPTION
**Pull request by anshikaa12**
>added slider component
>added README for the project

### CODE REVIEW

**Review by devananth — APPROVED** [review]
Good Work !

**Review by aryan-13 — APPROVED** [review]
Code looks good to merge!

File: documentation/documentation.html [hunk 1]
```diff
@@ -1925,7 +1928,25 @@ <h4 class="card-title">Apple iPhone 13- Blue</h4>
                 </div>
 
             </div>
-
+            <!-------------------------Slider system------------------------------->
+            <div id="slider-comp">
+                <div class="comp-intro">
+                    <h1 class="main-h1">Slider System</h1>
+                    <p class="main-p">TPixelUI provides responsive slider range</p>
```
Comment (Ayesha-7) [inline documentation/documentation.html:1935]: small typo here, *Pixel

File: documentation/documentation.html [hunk 2]
```diff
@@ -1925,7 +1928,25 @@ <h4 class="card-title">Apple iPhone 13- Blue</h4>
...
+                </div>
+                <h2 class="main-h2">Example of Standard Slider</h2>
+                <p class="doc-p">
+                    Use class <span class="prime-color-text">st-slider</span></p>
+                <div class="comp-container">
+                        <label for="volume" class="h3-text">Rating</label>
```
Comment (Ayesha-7) [inline documentation/documentation.html:1941]: here, label is for Volume but the text is Rating

**Review by Ayesha-7 — COMMENTED** [review]
Suggested some changes below. Other than that everything else is looking good. 👍
Suggestion: you can add some CSS to show at which element the user is/ which tab is active in the sidebar.

**Pull request merged**